docs/docs/topics/device-identity.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Device Authentication
```

Comment:
@desimone to note: Not sure if the title "Device Authentication" or the file name "device-identity" is more appropriate. Pick one, and I'll standardize.

Reply:
No perfect choice here but I like device identity.